edward/inferences/inference.py

```diff
@@ -194,6 +210,16 @@ def initialize(self, n_iter=1000, n_print=None, scale=None, logdir=None,
 
     if logdir is not None:
       self.logging = True
+
+      if logrun is None:
+        # Set default to timestamp
+        logrun = datetime.strftime(datetime.utcnow(), "%Y%m%dT%H%M%S")
```

Comment:
The T was a means of just separating the date and the time segments like most of the conventions here [https://www.loggly.com/docs/automated-parsing/#system](https://www.loggly.com/docs/automated-parsing/#system) and also removing all of the other non-numeric characters from the conventions.  However after using this in tensorboard a bit.  I'm going to replace the 'T' with a '_' just to keep it easier to read on the eye.